rpmbuild/copr-rpmbuild.spec

```diff
@@ -139,6 +139,12 @@ Requires: yum-utils
 %latest_requires ca-certificates
 %latest_requires distribution-gpg-keys
 %if 0%{?fedora}
+
+%if 0%{?fedora} >= 38
+%latest_requires dnf5
+%latest_requires dnf5-plugins
+%endif
+
 %latest_requires dnf
```

Comment:
Here we want to use `python3-dnf` I think.

Reply:
Updated